### DISCUSSION TITLE
Торговля у Пека

### DISCUSSION THREAD

**Discussion by dosinabox** [dosinabox/g2nr_unofficial_update#360] [Ideas]
Какие товары мог бы продавать Пек? 
- амулет Кантара
- косяки Борки
- тюк травы
- ?

_Originally posted by @dosinabox in https://github.com/dosinabox/g2nr_unofficial_update/issues/59#issuecomment-531552701_

**Comment by Kor-Angar** [comment]
- Мечи и арбалеты которые используют ополченцы + болты для арбалетов
- Конфискованный товар у Картера/Сары(?), в случае посадки
- Оружие заключенных (Нагур, Фернандо, Ренгару, Хальвор )
- Пояс ополченца (сейчас продаёт торговец на рыночной площаде)
- конфискованные "Особые рыбы" Хальвора, в случае посадки
- Конфискованные товары(мечи, сталь(?)) Беннета, 3 глава

**Comment by Kor-Angar** [comment]
так же наверное надо учесть лояльность Пека, т.е. если Гг сдал Пека Андре то Пек не будет продавать дурь, тюк травы, "особые рыбы".

**Comment by dosinabox** [comment]
> * Конфискованный товар у Картера/Сары(?), в случае посадки
> * Оружие заключенных (Нагур, Фернандо, Ренгару, Хальвор )
> * Конфискованные товары(мечи, сталь(?)) Беннета, 3 глава

По идее, он может продавать только то, от чего должен был избавиться. Это могут быть только нелегальные предметы - косяки Борки и тюк травы. При этом следует учесть, что тюк травы отдается Андрэ и вместе с этим проваливается квест Сифера. А теперь можно будет выкупить этот тюк и отнести его Сиферу. Что тогда делать с квестом?

Особые рыбы - спорно. Только в одной из них отмычки. Наверное, Пек может продавать предметы из этих рыб без самих рыб - кольцо, отмычки и кусок руды. Мол, сам проводил обыск заключенного при посадке.

Всё остальное (амулет Кантара, товары торговцев, оружие заключенных, сталь Беннета) находится на ответственном хранении и должно возвращаться своим хозяевам после выхода из тюрьмы. Ну, сейчас так) А в средних веках может и не так)

И ещё - как Пек получит предметы от заключенных, если они были посажены в тюрьму в то время, когда он находится в борделе? Это могут быть предметы Кантара, Сары, Нагура, Ренгару, Халвора и Фернандо. Предметы Борки, Беннета и тюк травы он получит уже вернувшись в казармы - с этим понятно.

**Comment by Kor-Angar** [comment]
> Особые рыбы - спорно. Только в одной из них отмычки. Наверное, Пек может продавать предметы из этих рыб без самих рыб - кольцо, отмычки и кусок руды. Мол, сам проводил обыск заключенного при посадке.

отличная идея

**Comment by Kor-Angar** [comment]
> Всё остальное (амулет Кантара, товары торговцев, оружие заключенных, сталь Беннета) находится на ответственном хранении и должно возвращаться своим хозяевам после выхода из тюрьмы. Ну, сейчас так) А в средних веках может и не так)

товары Кантара надо кому то отдать из торговцев на рынке. мне так кажется :-)

**Comment by Kor-Angar** [comment]
> По идее, он может продавать только то, от чего должен был избавиться. Это могут быть только нелегальные предметы - косяки Борки и тюк травы. При этом следует учесть, что тюк травы отдается Андрэ и вместе с этим проваливается квест Сифера. А теперь можно будет выкупить этот тюк и отнести его Сиферу. Что тогда делать с квестом?

квест Сифера не проваливать. Только тут нужно подумать об опыте. так как за отданный тюк Андре даёт больше опыта. С другой стороны Пеку может продавать тюк травы за приличную сумму.

**Comment by Kor-Angar** [comment]
может просто в случае посадки Кантара его товары передать на продажу Саре?

**Comment by Kor-Angar** [comment]
а то уже почти два года прошло :-) а воз и ныне там. С точки зрения баланса важны лишь товары Кантара, остальное это просто атмосферные моменты.
